tensor_api.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<ul><li>This should be in the binary element wise operation, as an example of automatic broadcasting.</li><li>Describe how automatic type casting works</li><li class="ql-indent-1">e.g. <code>a</code>  is a float tensor, what if it operate with a int / bool, ...</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/intel-isl/Open3D/pull/2067/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>